.github/pull_request_template.md

```diff
@@ -16,3 +16,10 @@
 ### Typescript
 - [ ] added or edited relevant Typescript type declarations
 - [ ] no type declaration updates needed
+
+## Merging
+For maintainers.
+
+To merge, select "Squash and Merge". Then:
+  1. Make sure the top commit message follows [Angular Git Commit Guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines),
+  2. Delete all other commit messages, but keep any lines designating [co-authors](https://docs.github.com/en/free-pro-team@latest/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors) so contributors will retain credit for their contributions.
```

Comment:
I think we should say something along the lines of this for step 2.
> 2. Clean up the description, make sure to keep any lines designating [co-authors] ...

Reply:
@smothers my problem with this is that I don't think it's obvious the commit messages are the "description" when it's filled with commit messages and you can't see the "description" place holder.

Maybe "Delete all other commit messages in the description, but keep any lines designating [co-authors](https://docs.github.com/en/free-pro-team@latest/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors) so contributors will retain credit for their contributions." ?

Reply:
resolved in latest